SU2_CFD/src/numerics_direct_mean_inc.cpp

```diff
@@ -334,7 +384,36 @@ void CCentJSTInc_Flow::ComputeResidual(su2double *val_residual, su2double **val_
       }
     }
   }
-  
+
+  /*--- Jacobian contributions due to grid motion ---*/
+  if (grid_movement) {
+
+    /*--- Recompute conservative variables ---*/
+
+    U_i[0] = DensityInc_i; U_j[0] = DensityInc_j;
+    for (iDim = 0; iDim < nDim; iDim++) {
+      U_i[iDim+1] = DensityInc_i*Velocity_i[iDim]; U_j[iDim+1] = DensityInc_j*Velocity_j[iDim];
+    }
+    U_i[nDim+1] = DensityInc_i*Enthalpy_i; U_j[nDim+1] = DensityInc_j*Enthalpy_j;
```

Comment:
I think you have all of the mean quantities computed, is it necessary to recompute here?

This might be a compressible vs incompressible difference but, in compressible flows only Energy gets affected by grid motion, is it completely equivalent to use Enthalpy here?

Reply:
One of the main changes to the governing equations for the incompressible solver is the simplified energy equation. The final term in the convective flux for the low Mach equations used in the incompressible solver is rho*Cp*T*v.  As part of the change to account for grid motion, the grid velocity is removed from all the velocity terms in the incompressible convective flux, similar to the compressible flow, which I think should still hold true with enthalpy.